lib/components/Note/Note.tsx

```diff
@@ -31,18 +31,18 @@ export const Note = ({
       aria-live="assertive"
```

Comment:
Was thinking should we have the `text-black` explicitly told to not rely on browser default?
(very sure that we have not text color explictly set in other components, but maybe we should)

Reply:
Added `text-black` class